src/k8s/pkg/k8sd/api/impl/k8sd.go

```diff
@@ -58,21 +60,50 @@ func GetClusterMembers(ctx context.Context, s *state.State) ([]apiv1.ClusterMemb
 		return nil, fmt.Errorf("failed to get cluster members: %w", err)
 	}
 
-	members := make([]apiv1.ClusterMember, len(clusterMembers))
+	members := make([]apiv1.NodeStatus, len(clusterMembers))
 	for i, clusterMember := range clusterMembers {
-		fingerprint, err := shared.CertFingerprintStr(clusterMember.Certificate.String())
```

Comment:
Is `fingerprint` no longer required?

Reply:
It was never. Just a leftover from early days where I randomly copied Microcluster examples.